install_config/install/advanced_install.adoc

```diff
@@ -399,12 +399,20 @@ specifications, and save it as *_/etc/ansible/hosts_*.
 
 === Multiple Masters, Multiple etcd, and Multiple Nodes
 
+There are two high availability methods Ansible supports. `pacemaker` and
```

Comment:
I think we should lead with the `native` method here something like

There are two high availability methods supported by the advanced installer: `native` and `pacemaker`. The former leverages the native HA capabilities built into OpenShift and can be combined with any load balancing solution. HAProxy will be configured as a load balancer if an `lb` group has been defined in the host inventory. If no `lb` group is specified, a pre-configured load balancer will be assumed.

The latter method configures Pacemaker, creating a Virtual IP on the active Master...